deploy/restartComponent.sh

```diff
@@ -22,9 +26,8 @@ for comp in $COMPONENTS; do
       exit 1
     fi
 
-    . /data/admin/wmagent/env.sh
     TAIL_LOG=`tail -n100 $COMPLOG`
-    $WMA_PATH/manage execute-agent wmcoreD --restart --components=$comp
+    $manage execute-agent wmcoreD --restart --components=$comp
```

Comment:
It is indeed better. It seems in the previous version this script must have been failing completely ....